drop-boxes/register-omero-metadata/backendinterface.py

```diff
@@ -194,16 +192,14 @@ def register_image_file_with_dataset_id(file_path, dataset_id, usr, pwd, host, p
         std_out, std_err = proc.communicate()
 
         if int(proc.returncode) == 0:
-
-            fist_line = std_out.splitlines()[0]
-            image_ids = fist_line[6:].split(',')
-
+            for line in std_out.splitlines():
+                if line[:6] == "Image:":
+                    image_ids = line[6:].split(',')
+                    break
```

Comment:
Can you please provide a comment here, what the terminal output looks like, to make this code more expressive.

Reply:
done